src/schematools/cli.py

```diff
@@ -578,7 +578,7 @@ def batch_validate(
                 )
             except (jsonschema.ValidationError, jsonschema.SchemaError) as struct_error:
                 errors[schema_file][meta_schema_version].append(
-                    f"{struct_error}: ({', '.join(struct_error.path)})"
+                    f"{struct_error}: {struct_error.path}"
```

Comment:
Or maybe cast elements to str: `.join(str(e) for e in struct_error.path)`?

Reply:
I considered that, but putting too long an expression in an f-string makes the code awkward to read. Also, this is unambiguous:

```python
>>> f"{1}" == f"{'1'}"
True
```

Not sure if this can happen, but I don't want to worry about it either :)